diff --git a/Makefile.in b/Makefile.in
index 73296f0..76cf776 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -69,10 +69,6 @@ realinstall:
 	# bash completion
 	install -m 0755 -d $(DESTDIR)/$(datarootdir)/bash-completion/completions
 	install -c -m 0644 src/bash_completion/fdns.bash_completion $(DESTDIR)/$(datarootdir)/bash-completion/completions/fdns
-	# apparmor profile
-	sh -c "if [ ! -d $(DESTDIR)/$(sysconfdir)/apparmor.d ]; then install -d -m 755 $(DESTDIR)/$(sysconfdir)/apparmor.d; fi;"
-	install -c -m 0644 etc/apparmor/usr.bin.fdns $(DESTDIR)/$(sysconfdir)/apparmor.d/.
-	install -c -m 0644 etc/apparmor/fdns-local $(DESTDIR)/$(sysconfdir)/apparmor.d/local/usr.bin.fdns
 
 install: all
 	$(MAKE) realinstall
